packages/fuels-macros/src/lib.rs

```diff
@@ -58,7 +58,7 @@ pub fn setup_program_test(input: TokenStream) -> TokenStream {
         .into()
 }
 
-#[proc_macro_derive(Parameterize, attributes(FuelsTypesPath, FuelsCorePath, NoStd))]
+#[proc_macro_derive(Parameterize, attributes(FuelsTypesPath, FuelsCorePath, NoStd, Ignore))]
```

Comment:
Where do we use this `Ignore` attribute ?

Reply:
`Tokenizable` also has it ^

Reply:
We mark fields that should not be `Tokenized` or `Parameterized`. A snippet from the struct-generating code:
```rust
            #(#[Ignore] pub #phantom_fields: #phantom_types, )*
```

Reply:
Thanks! Now I saw that `has_ignore_attr` looks for it.